vars/runCleanupResource.groovy

```diff
@@ -32,7 +32,7 @@ def call(Map params, String region){
     fi
 
     echo "Starting to clean resources ..."
-    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "k8s-local-kind-aws" || "$cloud_provider" == "k8s-local-kind-gce" ]]; then
+    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "k8s-local-kind-aws" || "$cloud_provider" == "k8s-local-kind-gce" || "$cloud_provider" == "aws-siren" || "$cloud_provider" == "gce-siren" ]]; then
```

Comment:
All such changes are not needed. Having "vars/getCloudProviderFromBackend.groovy" be updated is enough.
"cloud_provider" will give you "aws" or "gce". And these values are already part of the condition.
Existing check for 2 "k8s" values are redundant.

Reply:
fixed.